taskcluster/fenix_taskgraph/loader/single_dep.py

```diff
@@ -0,0 +1,76 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Ah I see, thanks! Although I don't see anything in `multi_dep` for the `only-for-attributes`, would I need to implement that for `multi_dep`?

Reply:
You're right. Yes, we have `only-for-build-types`: https://github.com/mozilla-mobile/fenix/pull/6923/files#diff-45e3065f7e5ef299faf52a26b40f0593R36

You can reuse the same logic, I think.

Reply:
So I couldn't use exactly the same logic. I did follow what you did for build-types though and I had to make some changes to the `group_by_*` function I made for the `only-for-attributes` setting.

I changed how the `groups` variable is setup. Instead of using the attribute (i.e. the build-type) as the entry, I use the `task.label` because I need to create one visual-metrics task per task that defines the run-visual-metrics attribute. I also don't use the multi_dep transform since it's incompatible with the visual-metrics tasks (it searches for some `job-symbol` that doesn't exist in these tasks). I added the deletion of the `dependent-tasks` entry to the end of the visual-metrics transformer.